### PR TITLE
Supermatter can now breed phoron, and phoron increases Radiation.

### PR DESCRIPTION
**Pull request by chaoko99**
🆑 
Methyl Bromide will now crank up the SM's phoron production if present in the chamber. It can be obtained from algae.
Phoron will now cause the SM to emit more radiation. Despite what your peers tell you, this was never a thing.
Supermatter monitor now gives readouts on the above features, and will now track Methyl Bromide
🆑

### CODE REVIEW

File: code/modules/supermatter/supermatter.dm [hunk 1]
```diff
@@ -500,14 +513,14 @@
...
+/obj/machinery/power/supermatter/proc/Consume(var/mob/living/user, var/amount = 200)
 	if(istype(user))
 		user.dust()
-		power += 200
+		power += amount
 	else
 		qdel(user)
 
-	power += 200
+	power += amount
```
Comment (scrdest) [inline code/modules/supermatter/supermatter.dm]: Setting the amount of power added per call param is a bit odd. Beats Random Magic Numbers though.

**Comment by chaoko99** [comment]
Honestly, methyl bromide is extremely difficult to mass produce in the space provided, it'd require cooperation to get made, which is why I chose it.

**Comment by chaoko99** [comment]
as for phoron, I believe that was an oversight or something because it seems to have used to do that.

File: nano/templates/supermatter_monitor.tmpl [hunk 2]
```diff
@@ -56,7 +56,33 @@
...
+			</div>
+			<div class="itemContent">
+				{{if data.SM_PHO = 100}}
+					<span class='bad'>{{:data.SM_PHO}}</span>
+				{{else data.SM_PHO > 50}}
+					<span class='average'>{{:data.SM_PHO}}</span>
+				{{else data.SM_PHO < 30}}
+					<span class='average'>{{:data.SM_EPR}}</span>
+				{{else}}
+					<span class='good'>{{:data.SM_EPR}}</span>
```
Comment (Naxodile) [inline nano/templates/supermatter_monitor.tmpl]:
```suggestion
					<span class='good'>{{:data.SM_PHO}}</span>
```
Pretty sure you don't want EPR values in your Phoron suppression display.

File: nano/templates/supermatter_monitor.tmpl [hunk 3]
```diff
@@ -56,7 +56,33 @@
...
+			<div class="itemLabel">
+				Crystal Resonance Factor
+			</div>
+			<div class="itemContent">
+				{{if data.SM_RAD >2}}
+					<span class='bad'>{{:data.SM_RAD}}</span>
+				{{else data.SM_RAD > 1}}
+					<span class='average'>{{:data.SM_RAD}}</span>
+				{{else}}
+					<span class='good'>{{:data.SM_EPR}}</span>
```
Comment (Naxodile) [inline nano/templates/supermatter_monitor.tmpl]:
```suggestion
					<span class='good'>{{:data.SM_RAD}}</span>
```
Same as above, incorrect values.

File: nano/templates/supermatter_monitor.tmpl [hunk 4]
```diff
@@ -56,7 +56,33 @@
...
+			<div class="itemLabel">
+				Crystal Ph Suppression
+			</div>
+			<div class="itemContent">
+				{{if data.SM_PHO = 100}}
+					<span class='bad'>{{:data.SM_PHO}}</span>
+				{{else data.SM_PHO > 50}}
+					<span class='average'>{{:data.SM_PHO}}</span>
+				{{else data.SM_PHO < 30}}
+					<span class='average'>{{:data.SM_EPR}}</span>
```
Comment (Naxodile) [inline nano/templates/supermatter_monitor.tmpl]:
```suggestion
					<span class='average'>{{:data.SM_PHO}}</span>
```
You'll want this one too if you're doing it through github exclusively.

Comment (chaoko99) [reply nano/templates/supermatter_monitor.tmpl]: Thanks.

**Comment by chaoko99** [comment]
I'm glad you have so much faith in me.

**Pull request merged**